cmake/DirectX-Headers.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
+exec_prefix="${prefix}"
+libdir="${prefix}/lib"
+includedir="${prefix}/include"
```

Comment:
You can't assume that CMAKE_INSTALL_LIBDIR=lib, it can be "lib64" or even "/usr/lib64" or "/opt/directx/lib". Meson handles this with `'${prefix}' / get_option('libdir')` which will prefix the libdir option with the literal pkg-config token `${prefix}` only if libdir is a relative path.

See e.g. https://github.com/jtojnar/cmake-snips/